doc/src/compute_hma.txt

```diff
@@ -0,0 +1,153 @@
+"LAMMPS WWW Site"_lws - "LAMMPS Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Section_commands.html#comm)
+
+:line
+
+compute hma command :h3
+
+[Syntax:]
+
+compute ID group-ID hma temp-ID keyword ... :pre
+
+ID, group-ID are documented in "compute"_compute.html command :l
+hma = style name of this compute command :l
+temp-ID = ID of fix that specifies the set temperature during canonical simulation :l
+keyword = {anharmonic} {u} {p Pharm} {cv} :l
+  {anharmonic} = compute will return anharmonic property values
+  {u} = compute will return potential energy
+  {p} = compute will return pressure.  the following keyword must be the difference between the harmonic pressure and lattice pressure as described below
+  {cv} = compute will return the heat capacity :pre
+:ule
+
+[Examples:]
+
+compute 2 all hma 1 u
+compute 2 all hma 1 anharmonic u p 0.9
+compute 2 all hma 1 u cv :pre
+
+
+
+[Description:]
+
+Define a computation that calculates the properties of a solid (potential
+energy, pressure or heat capacity), using the harmonically-mapped averaging
+(HMA) method. 
+This command yields much higher precision than the equivalent compute commands
+("compute pe"_compute_pe.html, "compute pressure"_compute_pressure.html, etc.)
+commands during a canonical simulation of an atomic crystal.
+
+
+In this method, the analytically known harmonic behavior of a crystal is removed from the traditional ensemble
+averages, which leads to an accurate and precise measurement of the anharmonic contributions without contamination 
+by noise produced by the already-known harmonic behavior. 
+A detailed description of this method can be found in ("Moustafa"_#hma-Moustafa). The potential energy is computed by the formula:
```

Comment:
I suggest that here at the start of the doc page you 
a) motivate more strongly why users should try this compute
b) list the main restrictions (noted at the bottom), namely solids, no diffusion, NVT intergration
So that people have a better sense of what this is about.  You can also note that
only one pair style is currently supported, but it is easy to add others.
Also, state whether manybody potentials could ever be supported?